src/calendar/README.md

```diff
@@ -0,0 +1,91 @@
+# @dojo/widgets/calendar/Calendar widget
+Dojo 2's `Calendar` renders a date grid with a dropdown month and year picker. The displayed month and year are controlled via properties, defaulting to the current day. Given the limitations of scaling date grid while maintaining usability, the calendar is not fully responsive at small screen sizes; it is instead recommended to switch to an alternative date picker for mobile.
+
+## Features
+
+- Renders a date grid with weekday headers, defaulting to the current month
+
+![Image of basic calendar](http://placekitten.com/450/300)
+
+- Clicking on the current month label opens a month picker that allows you to infinitely cycle through years and choose a month
+
+![Image of open month picker](http://placekitten.com/450/300)
+
+### Keyboard Usage
+- All controls are in the tab order and accessible by keyboard
+- Arrow keys move focus in the date grid, and navigating to a disabled date updates the month
+- Page up and page down move to the first and last date in the month
+- Arrow keys change the year within the month control popup
+
+### Accessibility Features
+- The month label is read by screen readers when updated through `aria-live: polite`
+- The month popup trigger, year control, month radios, and previous/next month arrows all have screen-reader-accessible labels and instructions
+- The weekdays and month radios are marked up with `<abbr>` tags that contain unabbreviated titles
+- Focus is controlled when opening and closing the month popup
+
+### i18n Features
+- Localized versions of `CalendarMessages`, `monthNames`, and `weekdayNames` can be passed in via properties.
+
+## Example Usage
+
+*Basic Example*
+```js
+import Calendar from '@dojo/widgets/calendar/Calendar';
+
+protected render: DNode () {
```

Comment:
I don't think we need to show the `import` of the module and a pseudocoded `render` function. This isn't valid code anyway, so why not just show the `w` call to keep boilerplate to a minimum? We also don't know if a user will be importing from the index module, this module, etc.